src/components/List/components/ListItem.tsx

```diff
@@ -17,13 +17,24 @@ export class ListItem<T = unknown> extends React.Component<ListItemProps<T>> {
     ref = React.createRef<HTMLDivElement>();
 
     render() {
-        const {item, style, sortable, sortHandleAlign, itemClassName, selected, active} =
-            this.props;
+        const {
+            item,
+            style,
+            sortable,
+            sortHandleAlign,
+            itemClassName,
+            selected,
+            active,
+            role = 'listitem',
+        } = this.props;
 
         return (
-            // eslint-disable-next-line jsx-a11y/click-events-have-key-events
             <div
-                role="listitem"
+                role={role}
+                tabIndex={
+                    (this.props.onClick || role === 'option') && !item.disabled ? 0 : undefined
```

Comment:
Item should never be tabbable

Reply:
I guess it should follow roving tabindex pattern